cob_hardware_config/cob4-2/config/base_joint_limits.yaml

```diff
@@ -1,31 +0,0 @@
-joint_limits:
```

Comment:
For the sake of completness, defining 
```
<xacro:property name="drive_vel" value="0.815" />
<xacro:property name="steer_vel" value="0.815" />
```
in any xacro within `cob_hardware_config` is superposed by the properties defined in the base.urdf.xacro - in fact the properties are nowhere to be seen as they get inserted (**--inorder**) directly!